admin-dev/themes/new-theme/js/app/pages/translations/components/app.vue

```diff
@@ -102,11 +109,7 @@
         });
         this.$store.currentDomain = '';
       },
-      onSave() {
-        this.$refs.principal.saveTranslations();
-        EventBus.$emit('hideModal');
-      },
-      onLeave() {
+      leavePage() {
```

Comment:
Please add a JSDoc header on your new functions.

It is important that each function describes:
- A short summary of what it does
- Type (and if possible, a description) of each parameter
- Type and description of what it returns
- Exceptions thrown and when

Documenting functions is important because
- It enables autocompletion by IDEs
- It eases code understanding

Example:
```js
/**
 * Assign the project to an employee.

 * @param {Object} employee - The employee who is responsible for the project.
 * @param {String} employee.name - The name of the employee.
 * @param {String} employee.department - The employee's department.
 *
 * @return {Project} Project instance
 */
Project.prototype.assign = function(employee) {
  // ...
}
```

You can find out more about JSDoc [here](http://usejsdoc.org/).